### PR TITLE
Update coronavirus research page for testing

### PR DESCRIPTION
**Pull request by DanielleThierryUSDSVA**
## Page to edit
url: va.gov/coronavirus-research/

## Origin of request (internal/stakeholder/user feedback)

Need to update page in staging for testing

## Description of what's needed (edits/link changes/additions)

Content updates

### CODE REVIEW

**Comment by DanielleThierryUSDSVA** [comment]
Hi @batemapf. Here's the PR for the landing page. It's failing to deploy latest changes. I'm not sure what the issue is from the message above. Any thoughts?

**Comment by batemapf** [comment]
I just reran the build again, same error. Looks like Jenkins cannot connect to the CMS for some reason. I tried connecting manually (https://prod.cms.va.gov/flags_list) and had no issue. Will try a clean PR and see if I get the same error just to be sure there isn't something weird going on with this specific change.

**Comment by batemapf** [comment]
Yep, same deal happened with an [empty PR](https://github.com/department-of-veterans-affairs/vagov-content/pull/715). @ncksllvn — any thoughts?

**Comment by DanielleThierryUSDSVA** [comment]
Thank you, @batemapf!

File: pages/coronavirus-research.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+**Note:** We won’t expose you to the virus that causes COVID-19 as part of any studies. If you do get sick during the study, we’ll work with you to make sure you get the care you need.
+
+#### After the study
+
+When the study is complete, we may ask you if we can contact you again for another study. You can say no if you’d prefer.
+
+We’ll also share the results of the study with you and your community. We won’t include your or any other participants’ name or any information that can be used to identify you.
+
+</div>
+</li>
```
Comment (kelsonic) [inline pages/coronavirus-research.md:124]: Not sure if these 2 lines (123, 124) are needed, might create unintended styling issues 🙂

Comment (DanielleThierryUSDSVA) [reply pages/coronavirus-research.md:124]: They're there to separate the accordions. But definitely let me know if they're not correct. Thanks!

**Comment by kelsonic** [comment]
This is what the page looks like on desktop:

localhost_3001_coronavirus-research_

This is what the page looks like on mobile:

localhost_3001_coronavirus-research_(Galaxy S5)

File: pages/coronavirus-research.md (hunk 1 above)
Comment (kelsonic) [reply pages/coronavirus-research.md:124]: They look great! 🎉

**Pull request merged**